frameworks/g-cloud-9/questions/services/otherWANs.yml

```diff
@@ -0,0 +1,27 @@
+name: "Other WANs"
+question: "List any other WANs your service is connected to."
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+list_item_name: "WANs"
+type: list
+
+validations:
+  -
+    name: answer_required
```

Comment:
If it's `optional: True` then you shouldn't need an `answer_required` validation.